docs/core/run-time-config/compilation.md

```diff
@@ -8,11 +8,112 @@ ms.topic: reference
 
 ## Tiered compilation
 
-- Configures whether the JIT compiler uses [tiered compilation](../whats-new/dotnet-core-3-0.md#tiered-compilation).
+- Configures whether the just-in-time (JIT) compiler uses [tiered compilation](../whats-new/dotnet-core-3-0.md#tiered-compilation).
 - In .NET Core 3.0 and later, tiered compilation is enabled by default.
 - In .NET Core 2.1 and 2.2, tiered compilation is disabled by default.
+- For more information, see the [Tiered compilation](https://devblogs.microsoft.com/dotnet/tiered-compilation-preview-in-net-core-2-1/) blog entry.
```

Comment:
This blog post is a bit dated. The current guide for TC is at https://github.com/dotnet/runtime/blob/master/docs/design/features/tiered-compilation-guide.md. It is also a bit dated but that's the one that'll be kept updated (hopefully soon). The guide references the blog post.